docker.local/bin/build.sharders.apple.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+GIT_COMMIT=$(git rev-list -1 HEAD)
+echo $GIT_COMMIT
+
+docker buildx build --platform linux/amd64 --build-arg GIT_COMMIT=$GIT_COMMIT -f docker.local/build.sharder/Dockerfile . -t sharder
```

Comment:
this should be a flag if the change is that trivial.

like if --m1 flag is set then linux/amd64

BTW should not be --apple, there are still many apple computers with intel chip

Reply:
Fixed